Closes #1: Removed Reliance on TDA, Everything should be configurable from Main

Chrisischris opened the pull request:
Closes #1

1tonyjiang reviewed (approved):
Seems to be working

Use command line args to determine api

Chrisischris commented:
@1tonyjiang look at this page to configure flake8 and black so your code passes test https://github.com/Chrisischris/algoTrader/wiki/Contributing

Chrisischris commented:
I reverted your changes on this branch and moved them to https://github.com/Chrisischris/algoTrader/tree/issue-3-Query_news_articles_related_to_a_stock_from_a_given_date so if you pull that branch and fix the flake8 issues and formatting (I also moved the twitter API keys to const.py so make sure that works) then open a PR
